docs/modeling/processes/variables.md

```diff
@@ -81,21 +85,60 @@ The following is an example payload for `POST /v1/process-instances` in the runt
 }
 ```
 
-## Mapping process variables
-Process variables can be mapped to and from variables in a BPMN element so that their values can be reused and updated throughout a process instance. 
+## Mapping variables
+Process variables in a process can be mapped to and from variables in BPMN elements such as [task variables](../forms/README.md#form-variables), [script variables](../scripts.md#script-variables) and [decision table values](../decisions.md). Input mapping is used to set the process variable sent from the process to the BPMN element and output mapping is used to set the target process variable to receive the results from the BPMN element after it has been executed.
```

Comment:
Does the script link exist (../scripts.md#script-variables)?

Reply:
This will exist when I submit the scripts PR, so not at the moment :-)